app/views/customers/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Customer</h1>
+
+<p>Name: <%= @customer.name %></p>
+<p>Items: </p>
+<ul>
+  <% @customer.items.each do |item| %>
+    <li>
+      Item Name: <%= item.name %> 
+       Price: <%= item.price %> 
+       Supermarket: <%= item.supermarket.name %>
+    </li>
+  <% end %>
+</ul>
+
+<%= form_with url: "/customers/#{@customer.id}", method: :post do |form| %>
```

Comment:
a more RESTful path to use here would be `"/customers/#{@customer.id}/items"` to indicate that you are creating an association between these 2 entities

Reply:
As we discussed in class today, "/customer_items" is also a RESTful path, but I usually opt for the path written above because it has the customer ID populated for easier access!